src/components/editor/editor-pane/editor-pane.tsx

```diff
@@ -99,15 +105,24 @@ export const EditorPane: React.FC<EditorPaneProps & ScrollProps> = ({ onContentC
   }, [editor, scrollState])
 
   const onBeforeChange = useCallback((editor: Editor, data: EditorChange, value: string) => {
+    if (value.length > maxLength && !maxLengthWarningAlreadyShown.current) {
```

Comment:
Can this trigger when one doesn't have edit rights?

Reply:
Yes. Because right now we have no information if a user is allowed to edit a doc or not.

Reply:
I'm unsure if the `onBeforeChange` event really will fire when the user doesn't change anything. I would guess it won't be triggered by "external" changes of the value.

Reply:
onBeforeChange will only be triggered if you type something.

Reply:
So correction: No. This won't trigger for read only access.